lib/node_modules/@stdlib/math/base/special/asinf/src/main.c

```diff
@@ -0,0 +1,137 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*
+* ## Notice
+*
+* The original C code, long comment, copyright, license, and constants are from [Cephes]{@link http://www.netlib.org/cephes}. The implementation follows the original, but has been modified according to project conventions.
+*
+* ```text
+* Copyright 1984, 1995, 2000 by Stephen L. Moshier
+*
+* Some software in this archive may be from the book _Methods and Programs for Mathematical Functions_ (Prentice-Hall or Simon & Schuster International, 1989) or from the Cephes Mathematical Library, a commercial product. In either event, it is copyrighted by the author. What you see here may be used freely but it comes with no support or guarantee.
+*
+* Stephen L. Moshier
+* moshier@na-net.ornl.gov
+* ```
+*/
+
+#include "stdlib/math/base/special/asinf.h"
+#include "stdlib/math/base/assert/is_nanf.h"
+#include "stdlib/math/base/special/sqrtf.h"
+#include "stdlib/constants/float32/half_pi.h"
+#include <stdint.h>
+
+/* Begin auto-generated functions. The following functions are auto-generated. Do not edit directly. */
+
+// BEGIN: poly_p
+
+/**
+* Evaluates a polynomial.
+*
+* ## Notes
+*
+* -   The implementation uses [Horner's rule][horners-method] for efficient computation.
+*
+* [horners-method]: https://en.wikipedia.org/wiki/Horner%27s_method
+*
+* @param x    value at which to evaluate the polynomial
+* @return     evaluated polynomial
+*/
+static float poly_p( const float x ) {
+	return 0.16666752422 + (x * (0.074953002686f + (x * (0.045470025998f + (x * (0.024181311049f + (x * 0.042163199048f)))))));
```

Comment:
Looks like there's a bug in `evalpoly-compile-c`. The first value is missing its suffix.

Reply:
Yes @kgryte . I looked into it. I believe, we need to add :

```C
else {
	horner += '{{dtype_suffix}}';
}
```

after line 124 at https://github.com/stdlib-js/stdlib/blob/55647661f551a753ba247511606eba2ee4766db8/lib/node_modules/%40stdlib/math/base/tools/evalpoly-compile-c/lib/main.js#L124

Reply:
Let me push a fix now.

Reply:
Pushed. The latest `develop` should have the patch which you can then rerun to get the correct polynomial function.

Reply:
@gunjjoshi You'll want to rerun the evalpoly script.

Reply:
Oh, that went out of my mind. Will do it now.